src/muxed_account.js

```diff
@@ -0,0 +1,155 @@
+import isString from 'lodash/isString';
```

Comment:
Could you keep this refactor, the separation into a different file, for another PR that is focused only on the refactor? As a reader it is difficult to see what has changed and what hasn't and to understand what is functional and what is refactor.

Reply:
Yep, done in 9fe601a5024ba47dd3eed8435bbe04ec469fc64c and 2f7da085bf49f2b2ebe682b1217f56dc543c2c40.